metrics/web/test/common/presentation/navigation/route_configuration/route_configuration_factory_test.dart

```diff
@@ -1,21 +1,43 @@
 // Use of this source code is governed by the Apache License, Version 2.0
 // that can be found in the LICENSE file.
 
-import 'package:metrics/common/presentation/navigation/constants/metrics_routes.dart';
+import 'package:metrics/common/presentation/navigation/route_configuration/route_configuration.dart';
 import 'package:metrics/common/presentation/navigation/route_configuration/route_configuration_factory.dart';
+import 'package:metrics/common/presentation/navigation/route_configuration/route_name.dart';
 import 'package:test/test.dart';
 
 void main() {
   group("RouteConfigurationFactory", () {
     const baseUrl = 'https://test.uri';
     const routeConfigurationFactory = RouteConfigurationFactory();
 
+    final loginRoute = RouteConfiguration.login();
+    final dashboardRoute = RouteConfiguration.dashboard();
+    final projectGroupsRoute = RouteConfiguration.projectGroups();
+    final debugMenuRoute = RouteConfiguration.debugMenu();
+
+    final isLoadingPageRoute = predicate<RouteConfiguration>(
```

Comment:
✅ 
It seems like the RouteConfiguration is Equatable so we can use `equals` in the test cases below. And as we know what URL we're passing to the factory - we know what resulting parameters should be if any. Therefore, in a single test case, we know the expected route configuration. Taking this into account, it seems like the predicates are redundant in the current test cases implementation. Moreover, most of them are used only once and can be initialized where used if necessary. This is related to the variables with routes as well.